addons/mail/static/src/core/common/composer.js

```diff
@@ -326,11 +326,19 @@ export class Composer extends Component {
                     ...props,
                     optionTemplate: "mail.Composer.suggestionPartner",
                     options: suggestions.map((suggestion) => {
-                        return {
-                            label: suggestion.name,
-                            partner: suggestion,
-                            classList: "o-mail-Composer-suggestion",
-                        };
+                        if (suggestion.isSpecial) {
```

Comment:
`special` is a very generic term that has 0 meaning. We should try to find a better-suited term.

Reply:
they are usually called commands or channel mentions but both in discuss refer to something else.
global mentions? group mentions?

Reply:
I don't have a better idea right now.
@alexkuhn is the name master, maybe he can find something better?

Reply:
I don't mind the name "special" mention, there's probably better but it's good enough.

I'd focus more effort on ensuring code of suggestion/mention is maintainable. The lack of abstraction makes it more complex than it should.

Reply:
I also feel like `special` is really generic and confusing seeing it everywhere. At some point, I would classify `@everyone` as a fix command just like `/help` `/leave` more than a `special mention`. And seems the autocomplete for everyone is not working correctly on runbot. when typing @, there is no suggestion for everyone until I type ever. not sure if this is on purpose or by mistake. for me @e should be enough for a suggestion already.

Reply:
@zel-odoo On other messaging apps, there's no suggestion for `@everyone` but they display the style of mention immediately on the composer. This is a feature that's intentionally hidden to prevent being used too often. People learn it from others using it.

In Odoo, we don't have (yet) the style of mention immediately on the composer, and we need to handle a inner-state for mentions. Hence why the tradeoff is to display the suggestion after typing `ever`. This is totally intentional.

The naming of `special` is good enough for me. I prefer we stick with this rather than struggle too long with another potential name or suggesting to refactor the whole code of mention / suggestion in this PR. We definitely want to clean the code, but not in this PR yet. This will be done in a follow-up PR.